src/js/actions/LibraryActions.js

```diff
@@ -71,6 +72,10 @@ export default {
             properties: ['openDirectory', 'multiSelections']
         }, (folders) => {
             if(folders !== undefined) {
+
+                folders.forEach((folder, index, array) => {
+                    array[index] = fs.realpathSync(folder);
```

Comment:
> Isn't Promisification more appropriate with API/Modules stuffs?

Yes, this is correct. The thing is that I'm not sure how well it works with `import` syntax.
You can create single promisified function higher in the scope via
```js
const realpathAsync = Promise.promisify(fs.realpath);
```